db/migrations/0014.change_course_sis_id_data_type.py

```diff
@@ -0,0 +1,10 @@
+#
+# file: migrations/0014.change_course_sis_id_data_type.py
```

Comment:
Is this change belongs to this issue? I thought the sis_id change was already merged in?

Reply:
This is a new issue, which I've documented here #117. I was thinking about opening a PR separately, but if anyone tests this PR on the spring/summer terms, it's going to break without this change.

Reply:
I guess I didn't see the harm in including it if it was documented separately.

Reply:
I think it's fine to include the resolution of #117 in this PR.  It's a small change and it fits closely with the other changes in this PR.  This is not a case where a lot of code for disparate issues is being forced into the same PR.

#117 is mentioned in the PR description and has been added to linked issues.  I think the one thing missing is to include it in the title of the PR.  The title mentions only #113, which might have confused @zqian.  Links to issues in PR titles should be complete or not done at all.

Reply:
Done!